uikit/form/FormHelperText/index.tsx

```
@@ -52,19 +61,21 @@ const FormHelperText = React.forwardRef<
     }
 
     &.disabled {
-      display: none;
+      ${!onErrorOnly && `display: none;`}
```

`FormHelperText` by default hides when the field is `disabled`.
It shouldn't do so if it's meant to show an error message.